src/main/drivers/compass/compass_ak8963.c

```diff
@@ -164,7 +164,7 @@ static bool deviceDetect(magDev_t * mag)
 
 bool ak8963Detect(magDev_t * mag)
 {
-    mag->busDev = busDeviceInit(BUSTYPE_ANY, DEVHW_AK8963, 0, OWNER_COMPASS);
+    mag->busDev = busDeviceInit(BUSTYPE_I2C, DEVHW_AK8963, mag->magSensorToUse, OWNER_COMPASS);
```

Comment:
If "BUSTYPE_I2C" instead of "BUSTYPE_ANY" is set here the AK8963 SPI compass will not be detected?

Reply:
@MJ666 oops, sorry. Will fix ASAP